### ISSUE TITLE
Add gesture based navigation

### ISSUE THREAD

**Issue by btopro** [elmsln/elmsln#947]
#684 voice support is _so_ earlier this week. We need gesture based navigational support so that people can wave their hands at the screen and have it navigate through the course. This is incredibly necessary for accessibility; I mean, what if someone is unable to speak and they don't understand how to use a keyboard, mouse or have a touch device.

https://github.com/mtschirs/js-objectdetect/ is just one example.

@michael-collins has other libraries I know he found the other day

**Comment by btopro** [comment]
@mmilutinovic1313 something fun to play with

**Comment by mmilutinovic1313** [comment]
https://github.com/mtschirs/js-objectdetect/ & http://stackoverflow.com/questions/10227464/how-to-detect-hand-gesture-in-live-webcam-using-javascript

Maybe add on sign language input in another "bucket of crazy fun task"

**Comment by btopro** [comment]
Make that a related issue though I'd be curious how that'd be helpful specifically. (open to it just can't envision it)

**Comment by btopro** [comment]
https://github.com/mtschirs/js-objectdetect best by far

**Comment by btopro** [comment]
http://mtschirs.github.io/js-objectdetect/examples/example_gesture_scroll.htm

**Comment by btopro** [comment]
all experiments cool but very buggy and nothing has graduated to a usable state
